go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/cosmos/gaia/v21
 
-go 1.22.6
+go 1.23.1
```

Comment:
Why is this necessary?

Reply:
pebbledb requires go 1.23 or higher

IMO the correct way to handle this is to bring this in at the next routine upgrade.

That will ensure that all of the validators compile with go 1.23 (and prevent any potential apphash mismatches or other oddities that can arise from mixed go runtimes)